### DISCUSSION TITLE
How do I set the background colour of the window when the page loads?

### DISCUSSION THREAD

**Discussion by Keithyumi** [eclipse-theia/theia#14438] [Q&A]
I replaced the index.html file in the lib/front folder to implement a custom IDE, the background is white until the page is fully rendered, making the user feel as if the program is stuck, how do I change the background colour to diminish this experience.
<img width="1681" alt="image" src="https://github.com/user-attachments/assets/960599e1-fd7c-48cd-ad03-3246e0e16a50">

**Comment by Keithyumi** [comment]
How can I control the timing of the browser window display?

**Comment by msujew** [comment]
@Keithyumi You might want to use a splash screen. See [here](https://github.com/eclipse-theia/theia/pull/13505).
